BUILD

```diff
@@ -1,3 +1,5 @@
+subinclude("///go//build_defs:go")
```

Comment:
This is necessary because `//:version` is subincluded in `package/BUILD`, before the subincludes in `parse.preloadsubincludes` are loaded.